ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -365,4 +358,69 @@ open class SideTabBar: UIView {
             delegate?.sideTabBar?(self, didSelect: item, fromTop: false)
         }
     }
+
+    public func overrideTokens(_ tokens: SideTabBarTokens?) -> Self {
+        overrideTokens = tokens
+        return self
+    }
+
+    var defaultTokens: SideTabBarTokens = .init()
+    var tokens: SideTabBarTokens = .init()
+    var overrideTokens: SideTabBarTokens? {
+        didSet {
+            updateSideTabBarTokens()
+        }
+    }
+
+    private class CustomSideTabBarItemTokens: TabBarItemTokens {
```

Comment:
Could you add a quick comment to this class explaining its purpose? In case it's confusing for future devs.